python/node_test.py

```diff
@@ -1,10 +1,11 @@
 # tests that correspond to the `src/doc.rs` rust api
 import tempfile
+import iroh.iroh_ffi
```

Comment:
what does this do?

Reply:
this is just a cleaner import for the type checker. actually, most of the static type checkers for python don't really like the way Maturin seems to want to set things up. exposing a submodule with wildcard imports is frowned upon, and since the type checkers don't actually run code, they seem to have trouble with that. this just makes the submodule import explicit.

Reply:
actually, right now, type checkers will _probably_ also fail an import like:

```python
from iroh import Iroh
```

Because of the wildcard import. However, if you silence that, then you still get suggestions etc 🤷?

Additionally, for some reason async static/class methods don't get proper type interference, so all the async class methods on Iroh just return Unknown, which is annoying. That's a mypy issue though, not an iron issue.